script.js

```diff
@@ -0,0 +1,118 @@
+//     Task 1
+
+let str = "true";
+let boo = false;
+let num = 17;
+let und = undefined;
+let nul = null;
+
+console.log(typeof str, typeof boo, typeof num, typeof und, typeof nul);
+
+//     Task 2
+
+let height = 15;
+let width = 20;
+
+if (height > width) {
+  console.log(height);
+} else {
+  console.log(width);
+}
+
+//     Task 3
+
+for (let i = 1; i <= 20; i++) {
+  if (i % 3 === 0) {
+    console.log(i);
+  }
+}
+
+//     Task 4
+
+let key = true;
+let documents = true;
+let pen = true;
+let apple = false;
+let orange = true;
+
+let shouldGoToWork = key && documents && pen && (apple || orange);
+console.log(shouldGoToWork);
+
+//     Task 5
+
+let numFB = prompt("Введите число");
```

Comment:
It is a good practice to be sure that you are working with number type, you can add '+' before 'prompt'. Also, you can add condition for letters. Just to improve it.